.circleci/config.yml-z

```diff
@@ -0,0 +1,144 @@
+version: 2.1
```

Comment:
it looks like the -z files are backups from before the rename? if that is right do we need to keep them?